.github/workflows/scripts/openblas-clone.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
why separate script is needed for this?

Reply:
We don't use  the openblas-clone.sh, but file remains. I will remove it from the repo after the run